tests/debugging/exploration/preload.py

```diff
@@ -1,3 +1,13 @@
+import os
+import sys
+
+
+# Make the tests/debugging folder available as a module. We don't want to
+# go as further back as tests/ because we are likely to override a folder with
+# a similar name from the target project.
```

Comment:
Can you update this comment to reference the actual bug, which is that dependencies get installed in edit mode, which makes their `tests` module available.


An alternative, we can be sure to add _our_ tests path at the start of the sys path?